nbconvert/preprocessors/execute.py

```diff
@@ -103,7 +104,18 @@ class ExecutePreprocessor(Preprocessor):
             """
             )
     ).tag(config=True)
-
+    
+    shutdown_kernel = Enum(['graceful', 'immediate'],
+        default_value='graceful',
+        help=dedent(
+            """
+            If `graceful` (default), then the kernel will given time to clean
+            up after executing all cells, e.g., to execute its `atexit` hooks.
+            If `immediate` or `now, then the kernel is signaled to immediately
```

Comment:
now reading the other comments, I see `now` is a leftover... so, just need to remove it from the docstring as well...